torq.q

```diff
@@ -567,27 +553,28 @@ sys:{[cmd]
 	@[{result:system x;.lg.o[`system;"successfully executed"];result};cmd;catcherror[cmd;]]
 	};
 
+// Load additional config files, check if the specific configuration environment variable exists and load the settings if it does
+loadaddconfig:{[envvar;cnfgpath]
+	evstring:string envvar;
```

Comment:
I'd probably scrap this and just do `string[envvar]` in the two log lines?

Reply:
I had tried this before and was running into an error will try it and reply below with the exact issue

Reply:
when trying to start a process in debug mode it gives this error
 `'type `
 `[5] /home/jpeter/torqrepo/deploy/torq.q:188: .lg.format:{[loglevel;proctype;proc;id;message] "|"sv string[(.proc.cp[];.z.h;proctype;proc;loglevel;id)],enlist(),message}`

Reply:
Hmm, what's the exact `.lg.o` line you have?

Reply:
nevermind, I didn't have `string env` bracketed so obviously it wouldn't work *facepalm*